### PR TITLE
Fix imports

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

**Comment by marusak** [comment]
I have a bug in there. I'll fix and push again.

**Comment by marusak** [comment]
Updated

File: src/pyfaf/storage/custom_types.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 import re
 from sqlalchemy import func
-import sqlalchemy.types as types
+from sqlalchemy import types
```
Comment (mkutlak) [inline src/pyfaf/storage/custom_types.py]: You can change it to `from sqlalchemy import func, types`.

File: src/bin/faf-migrate-db [hunk 2]
```diff
@@ -22,10 +22,11 @@ import inspect
 import os
 import argparse
 import logging
-from pyfaf.storage import migrations
-from pyfaf.common import get_connect_string
 from alembic.config import Config
 from alembic import command
+from pyfaf.storage import migrations
```
Comment (mkutlak) [inline src/bin/faf-migrate-db]: `from pyfaf.storage import GenericTable, getDatabase, migrations` ?

**Comment by marusak** [comment]
thanks @mkutlak, missed those. It is fixed now

**Pull request merged**